internal/website/gatherexamples/gatherexamples.go

```diff
@@ -0,0 +1,271 @@
+// Copyright 2019 The Go Cloud Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Command gatherexamples extracts all examples in a Go module into a JSON
+// object. This is used as input for building the Go CDK Hugo website.
+package main
```

Comment:
Could you add a section here documenting the ways in which the original source is mutated?